tests/grep_test.py

```diff
@@ -107,14 +107,23 @@ def _test_grep_color(file_config_files, capsys, *, args=()):
     ret = main(('-C', str(file_config_files.cfg), 'OHAI', *args))
     assert ret == 0
     out, _ = capsys.readouterr()
-    expected = (
-        '\033[1;34m{}\033[m'
-        '\033[36m:\033[m'
-        'f'
-        '\033[36m:\033[m'
-        '\033[1;31mOHAI\033[m\n'
-    ).format(file_config_files.output_dir.join('repo1'))
-    assert out == expected
+    expected = [
+        (
+            '\033[1;34m{}\033[m'
+            '\033[36m:\033[m'
+            '\033[35mf\033[m'
+            '\033[36m:\033[m'
+            '\033[1;31mOHAI\033[m\n'
+        ).format(file_config_files.output_dir.join('repo1')),
+        (
+            '\033[1;34m{}\033[m'
+            '\033[36m:\033[m'
+            'f'
+            '\033[36m:\033[m'
+            '\033[1;31mOHAI\033[m\n'
+        ).format(file_config_files.output_dir.join('repo1')),
+    ]
```

Comment:
this should probably be a set and not a list since it's being used for containment -- probably also doesn't need to be separately assigned and can be directly in the assertion

it would improve readability to assign `directory = ...` and substitute it in as an f-string

Reply:
Addressed review comments.

Reply:
@asottile Please take a look now.